pkg/serializer/jsonstream/compressor.go

```diff
@@ -69,15 +66,23 @@ type compressor struct {
 	repacks             int    // numbers of time we had to pack this payload
 	maxUnzippedItemSize int
 	maxZippedItemSize   int
+	maxPayloadSize      int
+	maxUncompressedSize int
```

Comment:
is it still needed to define these as package-level vars?

Reply:
It's now a field of compressor, github makes it confusing ;)

Reply:
It's now a field of compressor, github is lying to you ;)

Reply:
wow you're right, the diff threw me off 😮